### DISCUSSION TITLE
How to add browse back & forward buttons in title bar like Slack?

### DISCUSSION THREAD

**Discussion by Gumichocopengin8** [tauri-apps/tauri#7476] [Q&A] [answered]
Does Tauri support browse back & forward buttons feature by default? if so, how to do it?
Or Do I need to implement it in Frontend? 

<img width="395" alt="image" src="https://github.com/tauri-apps/tauri/assets/34010665/43406311-cf7a-4ffc-b3ef-cfdf31a726db">

**Comment by FabianLars** [comment, marked as answer]
>  Or Do I need to implement it in Frontend?

Yep.
